aiven_mysql_migrate/migration.py

```diff
@@ -89,9 +89,8 @@ def _stop_migration(self, signum, frame):
     def list_databases(self) -> List[str]:
         with self.source.cur() as cur:
             cur.execute(
-                "SELECT SCHEMA_NAME FROM INFORMATION_SCHEMA.SCHEMATA WHERE SCHEMA_NAME NOT IN ({format_dbs})".format(
-                    format_dbs=", ".join(["%s"] * len(self.ignore_dbs))
-                ), tuple(self.ignore_dbs)
+                f"SELECT SCHEMA_NAME FROM INFORMATION_SCHEMA.SCHEMATA "
```

Comment:
nit: Here and in places below f strings are redundant where no formatting is done (no variables to format)